scss/main.scss

```diff
@@ -5,8 +5,10 @@
 // directory of the repo: https://github.com/twbs/bootstrap
 // If you update Bootstrap be sure to remember to update the JS loaded from
 // Bootstrap's CDN.
-@import 'bootstrap-4.1.0/bootstrap';
 
-@import 'lib/cover';
+@import 'bootstrap-4.1.0/bootstrap';
 @import 'base/base';
-@import 'layouts/nav';
+
+//@import 'layouts/nav';
```

Comment:
@mrcrymnd Looks great!

Question: did you intend to include `layouts/nav` and `layouts/graphwidget` here?  You have the files checked in but they aren't linked to anything.

Reply:
oops fixed thank you